web/src/components/core/SectionSkeleton.jsx

```diff
@@ -23,19 +23,27 @@ import React from "react";
 import { Skeleton } from "@patternfly/react-core";
 import { _ } from "~/i18n";
 
-const SectionSkeleton = () => (
-  <>
-    <Skeleton
-      screenreaderText={_("Waiting")}
-      fontSize="sm"
-      width="50%"
-    />
-    <Skeleton
-      screenreaderText={_("Waiting")}
-      fontSize="sm"
-      width="25%"
-    />
-  </>
-);
+const SectionSkeleton = ({ numRows = 2 }) => {
+  const WaitingSkeleton = ({ width }) => {
+    return (
+      <Skeleton
+        screenreaderText={_("Waiting")}
+        fontSize="sm"
+        width={width}
+      />
+    );
+  };
+
+  return (
+    <>
+      {
+        Array.from({ length: numRows }, (_, i) => {
+          const width = i % 2 === 0 ? "50%" : "25%";
```

Comment:
Nice trick :P. 

BTW, In the future, when more sections get shaped, I'd like to have more kind of skeletons.